README.md

```diff
@@ -37,29 +37,73 @@
 
 # Requirements
 
+## Android
+
+Your project must be using the Gradle Build Tool.
+
 ## iOS
 
 Your project must be using Swift Package Manager or CocoaPods as dependency manager (or both). No support for Carthage.
 
 <sub>Note: this gem was only tested in macOS environments.</sub>
 
-## Android
+## Preview
 
-The Java/Kotlin code generator hasn't been implemented yet. Star and "watch" this project and check back in the future, or help us build it [here](https://github.com/rogerluan/arkana/issues/1).
+### Kotlin
 
-<img width="200" alt="image" src="https://user-images.githubusercontent.com/8419048/177968055-6774ad8e-2ef3-45ed-9e26-fbe73a783083.png">
+<details><summary>Click here to see the Kotlin preview</summary>
+<p>
 
-## Preview
+The image below shows how the auto-generated file looks like.
+
+<div align="center">
+  <img src="docs/kotlin-demo.png">
+</div>
+
+Usage using the example code above:
 
-<details><summary>Click here to show an image with the preview!</summary>
+```kotlin
+import com.arkanakeys.MySecrets
+
+// Designed with testability and DI in mind
+println(MySecrets.Global.someBooleanSecret)
+println(MySecrets.Global.someIntSecret)
+println(MySecrets.Global.mySecretAPIKey)
+
+// Simulating environment selection using a random boolean value
+val keys = if (Math.random() < 0.5) MySecrets.Dev else MySecrets.Staging
+println(keys.serviceKey)
+```
```

Comment:
@husseinala could you confirm whether this implementation is valid? It's just a sample code, but I mean, I asked ChatGPT to generate this lol I don't even know if it's valid syntax 😂

Reply:
Yep, it's valid and looks great! ChatGPT did a good job 😄

Reply:
haha sweet!! Thanks for reviewing it 😁